device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsDeviceTwin.java

```diff
@@ -191,6 +192,30 @@ protected AmqpsConvertToProtonReturnValue convertToProton(Message message) throw
         }
     }
 
+    @Override
+    protected MessageImpl buildSubscribeToDesiredPropertiesProtonMessage()
+    {
```

Comment:
Nice, I like this method. Can you reference this method in the iotHubMessageToProtonMessage method in the same class since it uses this logic, too?

Reply:
sure